apps/sync/js/sync-engine/syncengine.js

```diff
@@ -117,6 +118,7 @@ var SyncEngine = (function() {
             then(payloadDec => {
           return {
             id: record.id,
+            last_modified: record.last_modified,
```

Comment:
Please improve https://github.com/mozilla-b2g/gaia/blob/master/apps/sync/test/unit/sync-engine/syncengine_test.js#L167 to test for `expect(Object.keys(list.data[0]).sort()).to.deep.equal(['id', 'last_modified', 'payload'])`